src/QAT_RPC/qat_rpc/zmq/wrappers.py

```diff
@@ -1,13 +1,23 @@
+from enum import Enum
 from time import time
 from typing import Union
+from importlib.metadata import version
 
 import zmq
 from qat.purr.backends.echo import get_default_echo_hardware
 from qat.purr.compiler.config import CompilerConfig
 from qat.purr.compiler.hardware_models import QuantumHardwareModel
 from qat.purr.compiler.runtime import get_runtime
+from qat.purr.integrations.features import OpenPulseFeatures
 from qat.qat import execute_with_metrics
 
+class Messages(Enum):
```

Comment:
I think some of this should live in a `qat_rpc/utils.py` module or even a central `Server` class, as it's not ZMQ specific, and there's potentially going to be other rpc methods in the future.
OK for now though.